docs/tutorials/python/file_view.md

```diff
@@ -1,4 +1,138 @@
-# File Views
-[See the current available tutorial](../python_client.md#file-views)
+# FileViews
```

Comment:
NIT: Should the files with information/scripts for FileViews be named like `fileview` or `file_view`? Should that be kept consistent?